compatibility/update-versions.sh

```diff
@@ -6,7 +6,10 @@
 cd "$(dirname "$0")"
 eval "$(../dev-env/bin/dade-assist)"
 
-bazel run //versions:update-versions -- -o $PWD/versions.bzl
+# Note: not using Bazel run so we actually have access to local files to run
+# `git tag`
+bazel build //versions:update-versions
+bazel-bin/versions/update-versions -o $PWD/versions.bzl
```

Comment:
Perfectly happy to change it so just an fyi: `bazel run --run_under "cd $PWD &&" …` keeps the working directory.

Reply:
Done.